packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift

```diff
@@ -5,24 +5,24 @@
 import Foundation
 
 #if os(iOS)
-import Flutter
+  import Flutter
```

Comment:
The only change in non-example/test files are the whitespace changes in this file.  Does change actually need a pubspec version bump @stuartmorgan?

Reply:
Actually there's also removal of a semicolon, so it's not just whitespace.

Reply:
That's also unquestionably a no-op change though in terms of compile results.

I'm in favor of having no version change or CHANGELOG for this, as something that doesn't affect anything but our own development.

Reply:
(But I'm also fine with leaving it as-is)